README.md

```diff
@@ -51,15 +51,18 @@ This script will configure your system to install global npm packages without ha
 
 ### Yeoman not found during project generation
 
-Sometimes when running the generator, you will get an error indicating that `yeoman` is not installed and a prompt with a message like: `-bash: yo: command not found`. We've found that this is sometimes caused by not having the node modules binaries route correctly configured.
+Sometimes when running the generator, you will get an error indicating that `yeoman` is not installed and a prompt with a message like: `-bash: yo: command not found`. This could be due to having more than one node version installed, or not having the right `PATH`. Either way, here is how to fix this issue.
 
-The easiest way to fix it is by adding this line to the end of your `~/.bashrc`, `~/.zshrc` or the configuration script for your shell:
-- `export PATH=/usr/local/share/npm/bin:$PATH`
+NOTE: This solution works only if you have installed node using nvm (node version manager).
 
-If the above does not work, another possible way to fix it is the following:
-- Uninstall `yeoman`: `sudo npm remove -g yo`
-- Manually set your node path: `sudo npm config set prefix $NVM_DIR/versions/node/$(node--version)`
-- export the `NODE_PATH` variable: `export NODE_PATH=$NVM_DIR/versions/node/$(node --version)/lib/node_modules`
-- Reinstall it: `npm install -g yo`
+#### Step 1: Check if you have the right version of node setted
+It's a common issue having two or more versions of node. What you need to do is make sure that the version that is shown with the command `node --version` is the same as the one in your nvm directory (`nvm list`). If not, you can set it using the command `nvm use -version-` (e.g.: `nvm use v13.8.0`).
```

Comment:
Jus as a comment, we can define a `.nvmrc` file containing the desired version and `nvm use` without params will take that version. This way we don't need to know which version is the correct one.
https://github.com/nvm-sh/nvm#nvmrc